spec/system/support_interface/provider_sync_courses_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.feature 'See provider course syncing' do
+  include DfESignInHelpers
+
+  scenario 'User switches sync courses on Provider' do
+    given_i_am_a_support_user
+    and_a_provider_exists
+    when_i_visit_the_providers_page
+    then_i_see_that_the_provider_is_not_configured_to_sync_courses
+
+    when_i_click_on_a_provider
+    then_i_see_that_course_syncing_is_off
+
+    when_i_click_on_the_enable_course_syncing_button
+    then_i_see_that_course_syncing_is_on
```

Comment:
Not intended as a PR bomb, but... would it be nice to spec that the courses were indeed synced when the cron ran?

Reply:
:+1: I've had a go at this - invoked clock from the existing system spec (with a stubbed API response) and can now see the newly import course. Let me know if you had something else in mind!